ozaria/site/components/cinematic/PageCinematic/index.vue

```diff
@@ -45,8 +59,22 @@ module.exports = Vue.extend({
           timeout: 3000
         })
       }
+    },
+
+    handleSoundVolume () {
+      if (this.isSoundOn) {
```

Comment:
Is Howler a global sound manager?  Seems like this is something that should be registered outside of a component so that it can be loaded and run for all pages not just cinematics.  Instead of registering as part of a component we can create something that subscribes to the store and modifies the sound.

Reply:
It is a global sound manager. But it is only being used in the Cinematic system.